src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Handlers/ValidateContentContext.cs

```diff
@@ -7,13 +13,27 @@ public ValidateContentContext(ContentItem contentItem) : base(contentItem)
         }
 
         public ContentValidateResult ContentValidateResult { get; } = new ContentValidateResult();
+
     }
 
     public static class ValidateContentContextExtensions
     {
-        public static void Fail(this ValidateContentContext context, params string[] errors)
+
+        public static void Fail(this ValidateContentContext context, params ValidationResult[] errors)
         {
             context.ContentValidateResult.Fail(errors);
         }
+
+        public static void Fail(this ValidateContentContext context, string errorMessage, params string[] memberNames)
+        {
+            if (memberNames.Any())
```

Comment:
NullReferenceException if the params is not set

Reply:
Idem i think